src/modules/shortcut.ts

```diff
@@ -0,0 +1,202 @@
+import {
+	command,
+	default as CookiecordClient,
+	Module,
+	listener,
+} from 'cookiecord';
+import { Message, MessageEmbed, User } from 'discord.js';
+import { send } from 'process';
+import { BaseEntity, DeepPartial, ObjectType } from 'typeorm';
+import { Shortcut } from '../entities/Shortcut';
+import { BLOCKQUOTE_GREY, BLOCKQUOTE_GREY_NUMBER, TS_BLUE } from '../env';
+import { sendWithMessageOwnership } from '../util/send';
+
+// https://stackoverflow.com/a/3809435
+const LINK_REGEX = /https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)/;
+
+export class ShortcutModule extends Module {
+	constructor(client: CookiecordClient) {
+		super(client);
+	}
+
+	@listener({ event: 'message' })
+	async runShortcut(msg: Message) {
+		const { content } = msg;
+		const commandPart = content.split(' ')[0];
+		if (!commandPart.startsWith('!')) return;
```

Comment:
You can get the actual prefix like [so](https://github.com/cookiecord/cookiecord/blob/master/src/command/commandParser.ts#L15) instead of hard-coding it.